docs/how_to_integrate_nexmo.rst

```diff
@@ -0,0 +1,27 @@
+How to integrate ``Nexmo``?
```

Comment:
I think this can be just added in the `README` as a note to switch backends.

This doc is not needed until there are specific settings for the backend.

Reply:
Under which version, should I mention it as the latest version is already released?

Reply:
It uses `KEY` key in the `OPTIONS` setting for holding API key instead of `SID` which is used for `Twilio`.

Reply:
okay, can you also mention this as a note.